### PR TITLE
v107.1.0

### PR DESCRIPTION
**Pull request by codeman7**
# 107.1.0

This minor release we added support to Snackbar to inform users when it has disappeared. As well as, improvements around touch targets for Chips, alignment improvements to Dialogs and various other bug improvements.

## New features

In this release we added a new method to the `MDCSnackbarManagerDelegate` protocol to inform users when the snackbar disappeared.

```swift
extension MyCustomViewController: MDCSnackbarManagerDelegate {
  func snackbarDidDisappear() {
    print("The snackbar disappeared I should act accordingly")
  }
}
```

### Chips

* [MDCChipCollectionViewCell to correctly support increased touch target (#9816)](https://github.com/material-components/material-components-ios/commit/9b21ec994ba2b34b8143f00081d7fc0238a65777) (Yarden Eitan)
* [Update MDCChipCollectionViewFlowLayout to respect sectionInset (#9813)](https://github.com/material-components/material-components-ios/commit/8daed7f588366f64c172027c51697e7ee8e0eacb) (Bryan Oltman)

### Dialogs

* [Consider all actions alignments for horizontal layout (#9805)](https://github.com/material-components/material-components-ios/commit/594e477685268094783deab7522e479b463a8760) (Galia Kaufman)
* [Consider all actions alignments for vertical layouts (#9804)](https://github.com/material-components/material-components-ios/commit/b32c232339dcac5f2d6585d5990da094d017d3c0) (Galia Kaufman)

### FlexibleHeader

* [Subtract minimumHeaderViewHeight from fhv_accumulatorMax.](https://github.com/material-components/material-components-ios/commit/9f57fd45347ca22ec37094b9a372eed976ed1323) (shingzhao)

### Snackbar

* [Add 'snackbarDidDisappear' to MDCSnackbarManagerDelegate. (#9815)](https://github.com/material-components/material-components-ios/commit/ecbb85397f685dd369399db8f2fd625ff8a856a6) (Yarden Eitan)

### CODE REVIEW

**Review by galiak11 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
+# 107.1.0
+
+This minor release we added support to Snackbar to inform users when it has disappeared. As well as, improvements around touch targets for Chips, alignment improvements to Dialogs and various other bug improvements.
```
Comment (andrewoverton) [inline CHANGELOG.md:3]: This minor release includes additions to the `MDCSnackbarManagerDelegate` protocol, touch target improvements to Chips, alignment improvements to Dialogs, and various other bug fixes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+
+## New features
+
+In this release we added a new method to the `MDCSnackbarManagerDelegate` protocol to inform users when the snackbar disappeared.
```
Comment (andrewoverton) [inline CHANGELOG.md:7]: In this release we added a new method to the `MDCSnackbarManagerDelegate` protocol that informs the user when a snackbar disappears.

**Review by andrewoverton — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (codeman7) [reply CHANGELOG.md:3]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (codeman7) [reply CHANGELOG.md:7]: Done

**Pull request merged**